scripts/valhalla_build_config

```diff
@@ -17,6 +17,7 @@ config = {
     'user_agent': optional(str),
     'tile_url': optional(str),
     'tile_url_gz': optional(bool),
+    'concurrency': optional(int),
```

Comment:
unrelated change, but i realized it was an undocumented feature in tile building, so i added it